kong/clustering/services/sync/rpc.lua

```diff
@@ -39,13 +39,25 @@ function _M.new(strategy)
 end
 
 
-function _M:init_cp(manager)
-  local purge_delay = manager.conf.cluster_data_plane_purge_delay
+local function inc_sync_result(res)
+  return { default = { deltas = res, wipe = false, }, }
+end
 
-  local function gen_delta_result(res, wipe)
-    return { default = { deltas = res, wipe = wipe, }, }
+
+local function full_sync_result()
+  local deltas, err = declarative.export_config_sync()
+  if not deltas then
+    return nil, err
   end
 
+  -- wipe dp lmdb, full sync
+  return { default = { deltas = deltas, wipe = true, }, }
```

Comment:
```suggestion
  return gen_delta_result(deltas, true)
```

Otherwise remove the function above

Reply:
Talked offline, it is not a blocker.